src/slices/GiveThunk.ts

```diff
@@ -278,8 +276,12 @@ export const changeMockGive = createAsyncThunk(
       return;
     } finally {
       if (giveTx) {
-        segmentUA(uaData);
-
+        trackSegmentEvent(uaData);
+        trackGAEvent({
```

Comment:
maybe add the recipient as the event label here?

Reply:
Done